docs/examples/use_cases/pytorch/resnet50/pytorch-resnet50.rst

```diff
@@ -55,7 +55,7 @@ Usage
    positional arguments:
    DIR                         path(s) to dataset (if one path is provided, it is assumed to have subdirectories named "train" and "val"; alternatively, train and val paths can be specified directly by providing both paths as arguments)
 
-   optional arguments:
+   optional arguments (for the full list please check `APEx ImageNet example <https://github.com/NVIDIA/apex/tree/master/examples/imagenet>`_)
```

Comment:
Is it spelled APEx or is it a typo?

Reply:
I don't remember why I put it here like this. Changed to `Apex`.